src/webview-contract/webviewDefinitions/inspektorGadget.ts

```diff
@@ -1,6 +1,6 @@
 import { WebviewDefinition } from "../webviewTypes";
 
-export interface InitialState {}
+export type InitialState = object;
```

Comment:
💡 Idea:  Just something to take a quick peak, https://github.com/Azure/vscode-aks-tools/compare/main...Tatsinnit:vscode-aks-tools:fix/dependbot-packages-eslint if you want to port any of these.